spec/all_pages_spec.rb

```diff
@@ -51,6 +51,17 @@ def get_locale(path)
         end
       end
 
+      it 'has correctly configured system-status meta tag' do
+        doc.css('meta').each do |meta|
+          if meta[:name] == 'system-status'
```

Comment:
The way the monitor's written doesn't really prescribe either meta tag or a specific meta name, so I don't know that it's necessary for us to enforce that in a spec. At the same time, this is probably a simple approach to find the relevant tag in content.

I might propose to avoid looping and just find the one we're looking for?

```suggestion
        meta = doc.at_css('meta[name="system-status"]')
```

Reply:
I like that much better.